boards/arm/stm32mp157c_dk2/stm32mp157c_dk2.dts

```diff
@@ -11,7 +11,25 @@
 	model = "STMicroelectronics STM32MP157-DK2  board";
 	compatible = "st,stm32mp157c-dk2", "st,stm32mp15";
 	chosen {
+		zephyr,console = &usart3;
```

Comment:
A comment might be valuable here to explain this is valid only if CONFIG_UART_CONSOLE=y which is not the default config

Reply:
Better this way ?

Reply:
Better but not clear enough. And this setting is actually confusing.
What about commenting these 2 lines and and add following comment:
```
By default, zephyr console and shell are assigned to remoteproc.
To enable console and shell over uart, uncomment following lines
and set the correct config in stm32mp157c_dk2_defconfig
```

Reply:
This is indeed better, thank you for your feedback !

Reply:
(Added also in documentation mention of this device tree step in order to use UART console)